content/docs/user-guide/contributing/docs.md

```diff
@@ -81,36 +77,60 @@ Launch the server locally with:
 $ yarn develop
 ```
 
-This will start the server on the default port, `3000`. Visit
-`http://localhost:3000/` and navigate to the docs in question. This will also
-enable the Git pre-commit hook that will be formatting your code and
+This will start the server on the default port, `8000`. Visit
+`http://localhost:8000/` and navigate to the page in question. This will also
+enable the Git pre-commit hook that will be formatting and linting your code and
 documentation files automatically.
 
-### Debugging
+### All available commands
+
+To build the project and run it:
+
+- `yarn build` - build assets in the `public` directory (`gatsby build`).
+- `yarn develop` - run development server with hot reload (`gatsby develop`).
+- `yarn start` - run production static server over the `public` directory.
+
+If you change source code files, run tests:
+
+- `yarn test` - run tests.
```

Comment:
Isn't this one also enforced automatically upon submitting PRs? Maybe add that note on the top of the section so no need to repeat it in every group of commands.

Reply:
put a note on enforcment to the very top of this section to avoid duplication